server/entity/experience.go

```diff
@@ -0,0 +1,102 @@
+package entity
+
+var maxLevel = 21474836477
+
+type ExperienceManager struct {
+	level    int
+	progress float64
+	totalXP  int
+}
+
+func NewExperienceManager() *ExperienceManager {
+	return &ExperienceManager{
```

Comment:
You can do just ``&ExperienceManager{}``

Reply:
fix